sklearn/cluster/tests/test_bicluster.py

```diff
@@ -132,6 +132,24 @@ def test_spectral_biclustering(global_random_seed, csr_container):
                 _test_shape_indices(model)
 
 
+@pytest.mark.parametrize("est", (SpectralBiclustering(), SpectralCoclustering()))
+def test_non_regression(est):
+    X, _ = make_classification()
+    est.fit(X)
```

Comment:
I am not sure what to add to the non_regression test. Should I do something more elaborate? Should it also fail for one cluster?

Reply:
Updated in [48e5e28](https://github.com/scikit-learn/scikit-learn/pull/29980/commits/48e5e28427d8cdca7c3d2fd0383f33dd3ebc7c16) to resemble the one cluster test